common/app/controllers/EmailSignupController.scala

```diff
@@ -49,16 +50,15 @@ class EmailFormService(wsClient: WSClient) {
   def submit(form: EmailForm): Future[WSResponse] = {
     val maybeTriggeredSendKey: Option[Int] = listIdsWithTrigger.get(form.listId)
 
-    wsClient.url(Configuration.emailSignup.url).post(
+    val idAccessClientToken = Configuration.id.apiClientToken
+    wsClient.url(Configuration.id.consentEmailerApi)
+      .addHttpHeaders("X-GU-ID-Client-Access-Token" -> s"Bearer $idAccessClientToken")
```

Comment:
Isn't there an `IdentityClient` in frontend for this kind of thing?

Reply:
There is but this controller is in common so I don't think we would want to make it dependant on something in Identity. I think the pattern would be to bring the IdApiClient into common.

Reply:
Yeah, move the client into common if you have to.  Email sign up is moving closer to being `identity` these days anyway - maybe it should live there one day.